pkg/kubecost/totals.go

```diff
@@ -0,0 +1,367 @@
+package kubecost
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/kubecost/cost-model/pkg/log"
+	"github.com/patrickmn/go-cache"
+)
+
+// TODO can we use ResourceTotals for all things or do we need specific structs
+// like AllocationResourceTotals, AssetResourceTotals, etc.
```

Comment:
Any elaboration needed here for future work?

Reply:
Maybe I can elaborate a bit, but basically, it says it all—do we benefit from having one struct for each, so that allocation- or asset-specific things can be recorded. Seems like we're in the clear for now, but I might leave the comment, as an idea.